draft-ietf-gnap-core-protocol.md

```diff
@@ -1876,8 +1876,10 @@ uri (string)
 
 wait (integer)
 : RECOMMENDED. The amount of time in integer
-            seconds the client instance SHOULD wait after receiving this continuation
-            handle and calling the URI.
+            seconds the client instance MUST wait after receiving this continuation
```

Comment:
I suggest to omit "seconds" and use `The amount of time in seconds...`.

Another minor comment is that JSON doesn't have the integer datatype, it has the "number" datatype.

Reply:
The value is an integer even if the holder is a float. We don't want to encourage fractional second timeouts here.

Reply:
What you get right now is 

> The amount of time in integer seconds

and that sounds awkward to me.

I understand that you don't want to encourage fractional second timeuts. I just wanted to say is that there is no such data type as "integer" in JSON, there is a JSON number, it might be a minor (but important) detail if you want to be precise as much as possible, and use JSON data types.

Reply:
Right - the value is in **integer seconds**, and it is encoded as a **JSON number**. It reads fine to me, I've seen the construct "integer seconds" used plenty of times in similar works.